mod_reforged/hooks/skills/skill.nut

```diff
@@ -1,4 +1,7 @@
 ::Reforged.HooksMod.hook("scripts/skills/skill", function(q) {
+	q.m.IsDamagingPoise <- false;
+	q.m.IsStunningFromPoise <- false;
```

Comment:
`IsStunningFromPoise` should not be a property of a skill, but rather what happens when a character's Poise reaches 0 should be a property of the Poise attribute itself. Just as an example this could, in theory, then be modified by perks/traits/effects etc. which would say "when this character's poise reaches 0, he gets Dazed instead of Stunned".